packages/react-instantsearch-core/src/connectors/connectInfiniteHits.js

```diff
@@ -53,14 +54,24 @@ export default createConnector({
       };
     }
 
-    const { hits, page, nbPages } = results;
+    const { page, nbPages } = results;
+
+    const hitsWithPositions = addAbsolutePositions(
+      results.hits,
+      results.hitsPerPage,
```

Comment:
We should stay consistent either we destruct either we use the dot form.

Reply:
fixed here 52ebc802201bbbbf7a6dbb3c719802bd1347c8ad